lefthook.yml

```diff
@@ -0,0 +1,7 @@
+pre-commit:
+  commands:
+    prettier:
+      glob: "*.{js,jsx}"
+      run: prettier --write {staged_files} && git add {staged_files}
+    test:
+      run: pnpm run test
```

Comment:
One really nice thing about `lefthook` is that you don't need `pretty-quick` dependency anymore which removes a ton of additional dependencies without losing any functionality.